packages/office-ui-fabric-react/src/components/Panel/Panel.base.tsx

```diff
@@ -347,14 +348,21 @@ export class PanelBase extends BaseComponent<IPanelProps, IPanelState> implement
     defaultRender?: (props?: IPanelProps) => JSX.Element | null,
     headerTextId?: string | undefined
   ): JSX.Element | null => {
-    const { headerText } = props;
+    const { headerText, headerTextProps = {} } = props;
+    const headerTextNativeProps = getNativeProps<React.HTMLAttributes<HTMLDivElement>>(headerTextProps, divProperties, ['id', 'className']);
```

Comment:
I think our pattern on most components is only to strip out native props when they are pulled out from the top level props object. When it comes to nested (subcomponent) props objects we spread them while preserving all properties. (Related to my `IPanelHeaderTextProps` comment, I think we should be clear about the type and what is actually preserved in the spread.)